cypress/integration/2.composition/composition.creation_instructor.js

```diff
@@ -7,19 +7,24 @@ Cypress.on('uncaught:exception', (err, runnable) => {
 });
 
 describe('Instructor Creates Composition', () => {
-
     beforeEach(() => {
         cy.login('instructor_one', 'test');
         cy.visit('/course/1/');
+        cy.wait(500);
     });
 
     it('should create a composition as an Instructor', () => {
         cy.log('should check composition panel features');
-        cy.get('#homepage-create-menu').should('exist').click();
-        cy.get('#create-project-menu input[type="submit"]')
-            .eq(1).contains('Create Composition')
-        cy.get('#create-project-menu input[type="submit"]')
-            .eq(1).click();
+
+        cy.visit('/course/1/project/create/', {
+            method: 'POST',
+            body: {
+                project_type: 'composition'
+            }
+        });
+
+        cy.wait(500);
```

Comment:
Does Cypress have a `wait until` pattern?

Reply:
Yeah - you can use `wait` to wait for specific XHR requests to come through.... that's just, more work to set up that doesn't necessarily need to be done in this pull request: https://docs.cypress.io/api/commands/wait.html#Alias